Demo_1/index.html

```diff
@@ -0,0 +1,496 @@
+            });
+
+            factfinder.communication.ResultDispatcher.subscribe("suggest", function (suggestions) {
+                if (suggestions) {
+                    showOverlay(0);
+                } else {
+                    hideOverlay(0);
+                }
+            });
+
+            factfinder.communication.FFCommunicationEventAggregator.addBeforeDispatchingCallback(function (event) {
+                if (event.type === "search" || event.type === "filter" || event.type === "sort" || event.type === "clearFilter" || event.type === "advisor") {
+                    showOverlay(0, true);
+                }
+            });
+        });
+    </script>
+
+    <style>
+        [unresolved] {
+            opacity: 0;
+        }
+
+        * {
+            font-family: sans-serif;
+        }
+
+        html, body {
+            font-size: 14px;
+            transition: opacity 0.4s;
+            margin: 0;
+            padding: 0;
+        }
+
+        main {
+            display: block;
+            transition: all 0.3s;
+        }
+
+        body {
+            overflow-y: scroll;
+            padding-top: 150px;
+        }
+
+        header {
+            width: 100%;
+            height: 150px;
+            background: #0b97c4;
+            color: white;
+            position: fixed;
+            top: 0;
+            z-index: 99991;
+        }
+
+        .sortboxContainer {
+            height: 75px;
+            position: absolute;
+            left: 15px;
+        }
+
+        .sortboxContainer .sortBoxCaption {
+            box-sizing: border-box;
+        }
+
+        nav {
+            height: 75px;
+            background-color: #23282d;
+        }
+
+        #searchHeader {
+            font-size: 34px;
+            width: 100%;
+            display: flex;
+            justify-content: center;
+        }
+
+        ff-searchbox {
+            width: 50%;
+        }
+
+        ff-searchbox input {
+            width: 100%;
+            outline: none;
+            background: #0b97c4;
+            border: none;
+            color: white;
+            margin-top: 10px;
+            height: 55px;
+            font-size: 34px;
+            line-height: 35px;
+            border-bottom: 1px solid white;
+            text-align: center;
+        }
+
+        ff-searchbutton {
+            display: flex;
+        }
+
+        button {
+            cursor: pointer;
+            background: none;
+            outline: none !important;
+            border: none;
+        }
+
+        ff-campaign-feedbacktext {
+            width: 100%;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            flex-direction: row;
+        }
+
+        #rightColumn {
+            display: flex;
+            justify-content: center;
+            flex-direction: column;
+        }
+
+        span.rating {
+            color: gold;
+            font-size: 1.5rem;
+        }
+    </style>
```

Comment:
How about moving the styles to CSS file. Separate files for each component (continue the convention we already have - each component in one file) and some 'global' file for CSS that is not applicable for components but the document itself.

Reply:
Would be certainly an improvement. But from my point of view, everything which needs more than 5 minutes is not worth the effort for this old demos

Reply:
OK.